proto3-wire.cabal

```diff
@@ -75,17 +75,17 @@ test-suite tests
 
   build-depends:
       base >=4.9 && <=5.0
-    , bytestring >=0.10.6.0 && <0.12.0
+    , bytestring >=0.10.6.0 && <0.13.0
```

Comment:
Should the bounds really be duplicated for packages that are dependencies of the library and the tests? Easy to accidentally introduce incoherencies

Reply:
How so?

Reply:
Well, I accidentally forgot to update the bound here and then had trouble building the test elsewhere.
The test depends on the library and IIUC they're configured at the same time, so we actually get the intersection of bounds when running the test suite. 
If the test suite a looser bound it's ignored because the library one is applied, but if the test has a more narrow bound, then the library goes un-tested with some versions it builds for.

Reply:
> packages that are dependencies of the library and the tests? 

Huh this sentence makes no sense. I meant components that depend on the library